libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/LocationOverlay.kt

```diff
@@ -90,20 +90,23 @@ class LocationOverlay(private val mapView: CycleMapView) :
     }
 
     fun lockOnLocation() {
-        lockedOn = true
+        setEnableAutoStop(false)
     }
 
     fun hideButton() {
         button.visibility = View.INVISIBLE
     }
-
     override fun onTouchEvent(event: MotionEvent, mapView: MapView?): Boolean {
```

Comment:
Is it worth adding a comment to this method explaining what we're doing and why, perhaps linking to our Github issue and/or (if you raise one) the issue on `osmdroid`?  Potentially then we could rip out code if/when you or someone else fixes this in `osmdroid`.

Reply:
I'll add a link back to this issue. If the change does go into osmdroid, we would be able to remove this onTouchEvent entirely.